### ISSUE TITLE
Add PPS Device Tree Support

### ISSUE THREAD

**Issue by PowerBroker2** [Qengineering/Jetson-Nano-image#24]
Thanks for the image, I've tried it and works amazing!

I think it would be useful to a lot of people (including myself) if the kernel in this image included support for the PPS device tree. Here is a tutorial on how to do this: [link](https://msadowski.github.io/pps-support-jetson-nano/). I would do this on my own Nano, but I don't have an Ubuntu 18.04 machine to run the Jetson SDK and flash the Nano, so I'd like for it to come included in the SD image.

Would this be possible? I'd greatly appreciate it!

**Comment by Qengineering** [comment]
The image is downloaded and used by many people, targeted to the 'average' user. PPS is quite an 'advanced' subject. It requires quite a bit of technical knowledge. I'm afraid this is getting to niche for the average user. I also want to be able to provide support. I know from experience that these topics can generate an avalanche of support requests of no use to the average user. I hope you understand that I set the PPS tree to a low priority. If there are more requests, we can always reconsider.
So I'm afraid you'll have to find someone with an Ubuntu machine first.

**Comment by PowerBroker2** [comment]
> PPS is quite an 'advanced' subject. It requires quite a bit of technical knowledge. I'm afraid this is getting to niche for the average user

No, that's not true. The concept of a PPS signal is super simple and adding device tree support is supposedly pretty easy. Using the PPS signal once the device tree is present in the kernel is even easier, especially with other packages like GPSD, NTP, and Chrony.

> If there are more requests, we can always reconsider

[Lots of Jetson users see the utility in this feature...](https://forums.developer.nvidia.com/search?expanded=true&q=jetson%20nano%20%22pps%22)
